src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -558,9 +558,9 @@ private static Properties loadProperties(File file)
             throws CheckstyleException {
         final Properties properties = new Properties();
 
-        FileInputStream fis = null;
+        InputStream fis = null;
         try {
-            fis = new FileInputStream(file);
+            fis = Files.newInputStream(file.toPath());
```

Comment:
not very obvious but ... https://dzone.com/articles/fileinputstream-fileoutputstream-considered-harmful

Should be forbid usage of FileInputStream/FileOutputStream by import control ?

Reply:
Sounds fine to me.

Reply:
Done.